coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -82,8 +85,9 @@ def _parse_documentation_with_symbols(self, param_identifiers,
         for line in lines:
             if line.strip().startswith(param_identifiers[0]):
                 parse_mode = self.Parameter
-                splitted = line[len(param_identifiers[0]):].split(
-                    param_identifiers[1], 1)
+                param_offset = line.rfind(
+                    param_identifiers[0]) + len(param_identifiers[0])
+                splitted = line[param_offset:].split(param_identifiers[1], 1)
```

Comment:
does this belong to the pervious commit?
if not please state this inside the commit description why this needed adaption